epsagon/wrappers/http_filters.py

```diff
@@ -52,6 +53,22 @@ def is_blacklisted_url(url):
     return False
 
 
+def is_payload_collection_blacklisted(url):
+    """
+    Return whether the payload should be collected according to the blacklisted
+    urls list in the Trace.
+    :param url: url string
+    :return:  True if URL is blacklisted, else False
+    """
+    url = urllib.parse.urlparse(url).netloc
+    trace_blacklist_urls = trace_factory.get_trace().url_patterns_to_ignore
```

Comment:
1. maybe make url_patterns_to_ignore a set on init (or an empty set if None was given so you don't have to worry about None value
2. then this function can become:
```
return any(blacklist_url in url for blacklist_url in trace_blacklist_urls)
```